File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        if(nums.size()==0)return 0;
        sort(nums.begin(),nums.end());
        int current=1;
        int best=1;

        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]+1==nums[i+1]){
                current++;
                best=max(current,best);
            }
            else if(nums[i+1]>nums[i]+1){
                current=1;
            }
        }
        return best;
    }
};

File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-0.py
class Solution:
    def maxAscendingSum(self, nums: List[int]) -> int:
        mx = nums[0]
        cp = nums[0]
        for i in range(len(nums) - 1):
            if nums[i] < nums[i + 1]:
                mx += nums[i + 1]
            else:
                mx = nums[i + 1]
            cp = max(cp, mx)
        return cp